### ISSUE TITLE
Cannot build on Debian Sid with ldc2

### ISSUE THREAD

**Issue by russel** [atilaneves/reggae#29]
I just pull 52b43dc39e332993ca41cb6f331d4734715be87d which is tag: v0.5.16, and tried to build using Ninja and ldc2:
```
|> (export DMD=ldc2 ; bash -x ./bootstrap.sh ninja)
+ set -euo pipefail
+ '[' ninja '!=' '' ']'
+ BACKEND=ninja
+ COMP=ldc2
+ rm -rf bin
+ echo 'Compiling reggae'
Compiling reggae
+ ldc2 -ofbin/reggae -Isrc -Ipayload -Jpayload/reggae src/reggae/json_build.d src/reggae/reggae.d src/reggae/reggae_main.d src/reggae/dub/interop.d src/reggae/dub/json.d payload/reggae/backend/binary.d payload/reggae/backend/make.d payload/reggae/backend/ninja.d payload/reggae/backend/package.d payload/reggae/backend/tup.d payload/reggae/options.d payload/reggae/reflect.d payload/reggae/config.d payload/reggae/build.d payload/reggae/types.d payload/reggae/sorting.d payload/reggae/dependencies.d payload/reggae/range.d payload/reggae/buildgen.d payload/reggae/package.d payload/reggae/ctaa.d payload/reggae/file.d payload/reggae/rules/c_and_cpp.d payload/reggae/rules/common.d payload/reggae/rules/d.d payload/reggae/rules/dub.d payload/reggae/rules/package.d payload/reggae/core/package.d payload/reggae/core/rules/package.d payload/reggae/dub/info.d
+ cd bin
+ echo 'Running boostrapped reggae with backend ninja'
Running boostrapped reggae with backend ninja
+ ./reggae -b ninja --dc=ldc2 ..
[Reggae] Writing reggae source files
[Reggae] Writing reggae configuration
[Reggae] Writing dub configuration
[Reggae] Compiling metabuild binary dcompile
[Reggae] Compiling metabuild binary build.o
Couldn't execute ./dcompile --objFile=build.o --depFile=reggaefile.dep ldc2 -I/home/Checkouts/Git/Reggae -Isrc -g -debug /home/Checkouts/Git/Reggae/reggaefile.d src/reggae/config.d src/reggae/options.d src/reggae/buildgen_main.d src/reggae/buildgen.d src/reggae/build.d src/reggae/backend/package.d src/reggae/backend/binary.d src/reggae/package.d src/reggae/range.d src/reggae/reflect.d src/reggae/dependencies.d src/reggae/types.d src/reggae/ctaa.d src/reggae/sorting.d src/reggae/file.d src/reggae/rules/package.d src/reggae/rules/common.d src/reggae/rules/d.d src/reggae/rules/c_and_cpp.d src/reggae/core/package.d src/reggae/core/rules/package.d src/reggae/backend/ninja.d src/reggae/backend/make.d src/reggae/backend/tup.d src/reggae/dub/info.d src/reggae/rules/dub.d
in /home/Checkouts/Git/Reggae/bin/.reggae:
Could not compile with args:
ldc2 -I/home/Checkouts/Git/Reggae -Isrc -g -debug /home/Checkouts/Git/Reggae/reggaefile.d src/reggae/config.d src/reggae/options.d src/reggae/buildgen_main.d src/reggae/buildgen.d src/reggae/build.d src/reggae/backend/package.d src/reggae/backend/binary.d src/reggae/package.d src/reggae/range.d src/reggae/reflect.d src/reggae/dependencies.d src/reggae/types.d src/reggae/ctaa.d src/reggae/sorting.d src/reggae/file.d src/reggae/rules/package.d src/reggae/rules/common.d src/reggae/rules/d.d src/reggae/rules/c_and_cpp.d src/reggae/core/package.d src/reggae/core/rules/package.d src/reggae/backend/ninja.d src/reggae/backend/make.d src/reggae/backend/tup.d src/reggae/dub/info.d src/reggae/rules/dub.d -ofbuild.o -c
ldc2: Unknown command line argument '-debug'.  Try: '/usr/bin/ldc2 -help'
ldc2: Did you mean '-d-debug'?


bin.name: build.o, bin.cmd: ./dcompile --objFile=build.o --depFile=reggaefile.dep ldc2 -I/home/Checkouts/Git/Reggae -Isrc -g -debug /home/Checkouts/Git/Reggae/reggaefile.d src/reggae/config.d src/reggae/options.d src/reggae/buildgen_main.d src/reggae/buildgen.d src/reggae/build.d src/reggae/backend/package.d src/reggae/backend/binary.d src/reggae/package.d src/reggae/range.d src/reggae/reflect.d src/reggae/dependencies.d src/reggae/types.d src/reggae/ctaa.d src/reggae/sorting.d src/reggae/file.d src/reggae/rules/package.d src/reggae/rules/common.d src/reggae/rules/d.d src/reggae/rules/c_and_cpp.d src/reggae/core/package.d src/reggae/core/rules/package.d src/reggae/backend/ninja.d src/reggae/backend/make.d src/reggae/backend/tup.d src/reggae/dub/info.d src/reggae/rules/dub.d
```

**Comment by atilaneves** [comment]
`ldc2` and its command-line arguments aren't supported yet due to laziness on my part. Does `ldmd2` work?

**Comment by russel** [comment]
Ah. I recollect, I already knew I should use ldmd2 not ldc2. It works fine. I guess this issues should stay since it remains true?

**Comment by atilaneves** [comment]
Yeah, let's keep it open. Hopefully I'll get back to it.

**Comment by carun** [comment]
Stumbled up this issue. `ldmd2` worked just fine.

**Comment by kinke** [comment]
Fixed by #116 + #117.
